src/NosCore.Parser/Parsers/I18NParser.cs

```diff
@@ -25,18 +25,19 @@
 using System.Threading.Tasks;
 using NosCore.Core;
 using NosCore.Core.I18N;
+using NosCore.Dao.Interfaces;
 using NosCore.Data;
 using NosCore.Data.Enumerations;
 using NosCore.Data.Enumerations.I18N;
 using Serilog;
 
 namespace NosCore.Parser.Parsers
 {
-    public class I18NParser<TDto> where TDto : II18NDto, new ()
+    public class I18NParser<TDto, TPk> where TDto : II18NDto, new () where TPk : struct
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename class 'I18NParser' to match pascal case naming rules, consider using '18NParser'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)